catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/ImageMetacardType.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.transformer.nitf;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.codice.alliance.catalog.core.api.impl.types.IsrAttributes;
+import org.codice.alliance.catalog.core.api.impl.types.SecurityAttributes;
+import org.codice.alliance.transformer.nitf.common.NitfAttribute;
+import org.codice.alliance.transformer.nitf.common.NitfHeaderAttribute;
+import org.codice.alliance.transformer.nitf.image.GraphicAttribute;
+import org.codice.alliance.transformer.nitf.image.ImageAttribute;
+import org.codice.alliance.transformer.nitf.image.LabelAttribute;
+import org.codice.alliance.transformer.nitf.image.SymbolAttribute;
+import org.codice.alliance.transformer.nitf.image.TextAttribute;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.impl.MetacardTypeImpl;
+import ddf.catalog.data.impl.types.AssociationsAttributes;
+import ddf.catalog.data.impl.types.ContactAttributes;
+import ddf.catalog.data.impl.types.DateTimeAttributes;
+import ddf.catalog.data.impl.types.LocationAttributes;
+import ddf.catalog.data.impl.types.MediaAttributes;
+import ddf.catalog.data.impl.types.ValidationAttributes;
+
+public class ImageMetacardType extends MetacardTypeImpl {
+    private static final String NAME = "isr.image";
+
+    public ImageMetacardType() {
+        super(NAME, (Set<AttributeDescriptor>) null);
+        getDescriptors();
+    }
+
+    private void getDescriptors() {
+        descriptors.addAll(BasicTypes.BASIC_METACARD.getAttributeDescriptors());
+        descriptors.addAll(getDescriptors(GraphicAttribute.values()));
+        descriptors.addAll(getDescriptors(ImageAttribute.values()));
+        descriptors.addAll(getDescriptors(LabelAttribute.values()));
+        descriptors.addAll(getDescriptors(SymbolAttribute.values()));
+        descriptors.addAll(getDescriptors(TextAttribute.values()));
+        descriptors.addAll(getDescriptors(NitfHeaderAttribute.values()));
+        descriptors.addAll(new AssociationsAttributes().getAttributeDescriptors());
+        descriptors.addAll(new ContactAttributes().getAttributeDescriptors());
```

Comment:
Question without digging too deep into this, do the CoreAttributes ultimately end up  on the metacardtype?